shared/src/IWritablePropertyResponse.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Shared
+{
+    /// <summary>
+    /// The interface that defines the structure of a writable property response.
```

Comment:
this is relevant for scenario where a device responds to a property update request sent by service:
-> service -> update twin patch -> device subscribed to twin property updates -> device receives property update -> device responds to the property update request

In the "device responds to the property update request", for devices following pnp convention, the patch should follow the following format:
https://docs.microsoft.com/en-us/azure/iot-pnp/concepts-convention#writable-properties